api-catalog-ui/frontend/src/components/Wizard/DialogDropdown.jsx

```diff
@@ -53,7 +53,11 @@ export default class DialogDropdown extends Component {
     }
 
     renderDropdown() {
-        if (!this.props.visible || !Array.isArray(this.state.data)) {
+        if (
+            !this.props.visible ||
+            !Array.isArray(this.state.data) ||
+            (process.env.REACT_APP_API_PORTAL !== undefined && process.env.REACT_APP_API_PORTAL === 'true')
```

Comment:
is the verification for undefined needed in this case?

Reply:
Fixed